app/controllers/report_controller/reports/editor.rb

```diff
@@ -525,7 +525,8 @@ def gfv_model
         @edit[:new][:tz] = session[:user_tz]
         build_perf_interval_arrays(@edit[:new][:perf_interval]) # Build the start and end arrays for the performance interval chooser
       end
-      if model_report_type(@edit[:new][:model]) == :chargeback
+      if model_report_type(@edit[:new][:model]).to_s.starts_with?("chargeback")
+        @edit[:new][:cb_model] = @edit[:new][:model] == "ChargebackVm" ? "Vm" : "ContainerProject"
```

Comment:
@zeari it seems that this may deserve a table (so it's more explicit for the future rather than a ternary operator). Something like:

```ruby
REPORT_MODELS = {
  "ChargebackVm"               => "Vm",
  "ChargebackContainerProject" => "ContainerProject"
}
```

Or maybe a method (with an even more meaningful name than the one I found):

```ruby
def report_cb_model(model)
    model.gsub(/^Chargeback/, "")  # Or anything equivalent that is nicer
end
```